src/main/java/org/broadinstitute/hellbender/tools/walkers/bqsr/AnalyzeCovariates.java

```diff
@@ -477,13 +477,7 @@ private File resolveCsvFile() {
         if (csvFile != null)  {
             return csvFile;
         } else {
-            try {
-              final File result = File.createTempFile("AnalyzeCovariates", ".csv");
-              result.deleteOnExit();
-              return result;
-            } catch (IOException e) {
-                throw new UserException("Could not create temporary Csv file",e);
-            }
+            return BaseTest.createTempFile("AnalyzeCovariates", ".csv");
```

Comment:
@droazen should we just remove the `BaseTest.createTempFile()` method entirely?  Why not just have tests call the IOUtils one?